webpack.config.js

```diff
@@ -41,7 +41,7 @@ const sassRules = [
   }
 ];
 
-module.exports = ({ production } = {}, { extractCss, analyze, tests, hmr, port, host } = {}) => ({
+module.exports = ({ production, extractCss, analyze, tests, hmr, port, host, } = {}) => ({
```

Comment:
Old way of passing eg. `--extractCss` not recognized by webpack5 anymore.
I tested to pass `--env extractCss` and here `extractCss` logged to `true`.
This explains the reduction of 2 params to 1 param